tests/polars/test_polars_container.py

```diff
@@ -1,6 +1,7 @@
 # pylint: disable=redefined-outer-name
 """Unit tests for Polars container."""
 
+import typing
 from typing import Optional
```

Comment:
```suggestion
from typing import Annotated, Optional
```
And remove
```python
try:
    from typing import Annotated  # type: ignore
except ImportError:
    from typing import Annotated  # type: ignore
```
below?